examples/pruning/keras_integration.py

```diff
@@ -0,0 +1,94 @@
+"""
+Optuna example that demonstrates a pruner for Keras.
+
+In this example, we optimize the validation accuracy of hand-written digit recognition using
+Keras and MNIST, where the architecture of the neural network is optimized. Throughout the
+training of neural networks, a pruner observes intermediate results and stops unpromising trials.
+
+You can run this example as follows:
+    $ python keras_integration.py
+
+"""
+
+import optuna
+from optuna.integration import KerasPruningCallback
+
+import keras
+from keras.datasets import mnist
+from keras.layers import Dense
+from keras.models import Sequential
+from keras import optimizers
+from keras import regularizers
+
+
+BATCHSIZE = 128
+CLASSES = 10
+EPOCHS = 20
+
+
+def model_fn(trial):
+    # Model
+    n_layers = trial.suggest_int('n_layers', 1, 3)
+    weight_decay = trial.suggest_uniform('weight_decay', 1e-10, 1e-3)
```

Comment:
[The official example](https://github.com/keras-team/keras/blob/master/examples/mnist_mlp.py#L38) in keras-team repository uses dropout instead of weight decay. I think we can optimize the dropout rate of each layer by using `trial.suggest_uniform`, and it will show the strength of Optuna's define-by-run API.
What do you think of it?

Reply:
Alright. Dropout works better than weight_decay usually, so I guess dropout in the example would be better.